Source/Scripts/carthage-bash-completion

```diff
@@ -49,7 +50,7 @@ _carthage() {
     fi
 
     case "${prev}" in
-        --output)
+        --output|--project-directory)
```

Comment:
It might be better to use `_filedir -d` for `--project-directory` if it accepts only a directory.

```
        --project-directory)
            _filedir -d
            return 0
            ;;
```